### ISSUE TITLE
oracle delegation signature

### ISSUE THREAD

**Issue by marc0olo** [aeternity/aepp-sdk-js#1437]
again for aepp-sophia-examples. the tests used to work and now fail for some reason after updating to v11.0.0:

```sh
1) OracleDelegation
       Interact with the contract
         should register an oracle:
     NodeInvocationError: Invocation failed: "Error in oracle_register: bad_signature"
      at l (node_modules/@aeternity/aepp-sdk/dist/aepp-sdk.js:1:94920)
      at Object.c.call (node_modules/@aeternity/aepp-sdk/dist/aepp-sdk.js:1:97516)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
      at async Object.c._estimateGas (node_modules/@aeternity/aepp-sdk/dist/aepp-sdk.js:1:95715)
      at async Object.c.call (node_modules/@aeternity/aepp-sdk/dist/aepp-sdk.js:1:97636)
      at async Context.<anonymous> (test/oracleDelegation.js:47:13)

  2) OracleDelegation
       Interact with the contract
         should extend an oracle:
     NodeInvocationError: Invocation failed: "Error in oracle_extend: bad_signature"
      at l (node_modules/@aeternity/aepp-sdk/dist/aepp-sdk.js:1:94920)
      at Object.c.call (node_modules/@aeternity/aepp-sdk/dist/aepp-sdk.js:1:97516)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
      at async Object.c._estimateGas (node_modules/@aeternity/aepp-sdk/dist/aepp-sdk.js:1:95715)
      at async Object.c.call (node_modules/@aeternity/aepp-sdk/dist/aepp-sdk.js:1:97636)
      at async Context.<anonymous> (test/oracleDelegation.js:53:13)

  3) OracleDelegation
       Interact with the contract
         should respond to a specific query:
     NodeInvocationError: Invocation failed: "oracle not valid"
      at l (node_modules/@aeternity/aepp-sdk/dist/aepp-sdk.js:1:94920)
      at Object.c.call (node_modules/@aeternity/aepp-sdk/dist/aepp-sdk.js:1:97516)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
      at async Object.c._estimateGas (node_modules/@aeternity/aepp-sdk/dist/aepp-sdk.js:1:95715)
      at async Object.c.call (node_modules/@aeternity/aepp-sdk/dist/aepp-sdk.js:1:97636)
      at async Context.<anonymous> (test/oracleDelegation.js:58:39)
 ```

**Comment by marc0olo** [comment]
did we introduce this problem with https://github.com/aeternity/aepp-sdk-js/commit/563a9722baa15a60110a61fb21088e9158442518 ?

do I just need to change the tests here? https://github.com/aeternity/aepp-sophia-examples/blob/bump-aeproject-and-sdk-versions/test/oracleDelegation.js

I am not sure what @davidyuk did there to be honest and what's the reason for removing the `onAccount` usage

**Comment by subhod-i** [comment]
> do I just need to change the tests here? https://github.com/aeternity/aepp-sophia-examples/blob/bump-aeproject-and-sdk-versions/test/oracleDelegation.js
> 
> I am not sure what @davidyuk did there to be honest and what's the reason for removing the `onAccount` usage

This change has no effects. `onAccount` has been removed because it gets included using `this.address()` method anwyay and even the signing buffer order is preserved too.

**Comment by davidyuk** [comment]
was fixed in #1462
